pkg/utils/name_transformers.go

```diff
@@ -34,6 +34,20 @@ func UpperCamel(name string) string {
 	return strings.Join(parts, "")
 }
 
+func LowerKebab(name string) string {
```

Comment:
It doesn't look like this function or `UpperCamel` are actually used because name transformation is happening on the server. `DotNETNameTransform` (via `MapToDotNETJSONFormat) and `MapToEnvFormat` are used with mount formats via `SecretsToBytes` but it looks like that's exclusively for custom formats so you shouldn't need to add one of those

Reply:
Ah, okay. This will actually force the change @Piccirello was mentioning as well since we already have the behavior he mentions serverside.